src/base/misc/fatfs.c

```diff
@@ -506,7 +506,7 @@ int read_boot(fatfs_t *f, unsigned char *b)
 
     build_boot_blk(f, b);
 
-    memcpy(b + 0x03, "DOSEMU10", 8);
+    memcpy(b + 0x03, "IBM  3.3", 8);
```

Comment:
I can see why you might want to preserve the DOSEMU10 value for mkfatimage16, but for fatfs.c the value only exists in memory, or is there something I'm missing here?